mwcleric/auth_credentials.py

```diff
@@ -23,6 +23,16 @@ def __init__(self, username=None, password=None, user_file=None):
         :param password: Password, this is the actual value of the password, not the "name" of a "bot password"
         :param user_file: Either a file or a system variable as a nicknamed account to look for
         """
+        if start_over:
+            if user_file is None:
+                raise ValueError('No user_file is defined.')
+            start_over_confirm = input('Are you sure you want to overwrite your previous credentials? [Y]es or [N]o ')
+            if start_over_confirm.lower()[0] == 'y':
+                account_data = self.prompt_user_info()
+                self.save_account_data(account_data, user_file)
+            else:
+                pass
```

Comment:
Can you add a comment here explaining that we're going to proceed with the login as normal, assuming that the start_over param was a mistake?